schemes/zhang_mcfarlane/zm_conv_convtran.meta

```diff
@@ -18,17 +53,18 @@
   dimensions = ()
   intent = in
 [ doconvtran ]
-  standard_name = flag_for_zhang_mcfarlane_deep_convective_transport?
+  standard_name = flag_for_tracer_transport_by_zhang_mcfarlane_deep_scheme
   units = flag
   type = logical
-  dimensions = (1:number_of_ccpp_constituents)
+  dimensions = (number_of_ccpp_constituents)
   intent = in
 [ q ]
   standard_name = ccpp_constituents
   units = none
   type = real | kind = kind_phys
-  dimensions = (1:horizontal_loop_extent,1:vertical_layer_dimension,1:number_of_ccpp_constituents)
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension,number_of_ccpp_constituents)
   intent = in
+  advected = true
```

Comment:
Oops, should've clarified. `advected = true` is only for individual constituents (not the whole array). I imagine this "works" since you've tested it, but I'd remove this to avoid future confusion

Reply:
removed advected from this location